--- work/cronie-1.5.4.orig/src/crontab.c	2019-03-15 10:43:05.000000000 +0100
+++ work/cronie-1.5.4/src/crontab.c	2019-10-27 10:48:09.000000000 +0100
@@ -210,12 +210,6 @@
 static void parse_args(int argc, char *argv[]) {
 	int argch;
 
-	if (!(pw = getpwuid(getuid()))) {
-		fprintf(stderr, "%s: your UID isn't in the passwd file.\n",
-			ProgramName);
-		fprintf(stderr, "bailing out.\n");
-		exit(ERROR_EXIT);
-	}
 	if (strlen(pw->pw_name) >= sizeof User) {
 		fprintf(stderr, "username too long\n");
 		exit(ERROR_EXIT);
