--- git-2.19.2/daemon.c.orig	2018-11-21 15:58:11.000000000 +0100
+++ git-2.19.2/daemon.c	2019-03-11 22:05:56.000000000 +0100
@@ -110,7 +110,7 @@
 {
 	va_list params;
 	va_start(params, err);
-	logreport(LOG_ERR, err, params);
+//	logreport(LOG_ERR, err, params);
 	va_end(params);
 }
 
@@ -121,13 +121,13 @@
 	if (!verbose)
 		return;
 	va_start(params, err);
-	logreport(LOG_INFO, err, params);
+//	logreport(LOG_INFO, err, params);
 	va_end(params);
 }
 
 static void NORETURN daemon_die(const char *err, va_list params)
 {
-	logreport(LOG_ERR, err, params);
+//	logreport(LOG_ERR, err, params);
 	exit(1);
 }
 
@@ -1431,10 +1431,14 @@
 			log_destination = LOG_DESTINATION_STDERR;
 	}
 
+/*
+        no syslog on QNX NTO
+
 	if (log_destination == LOG_DESTINATION_SYSLOG) {
 		openlog("git-daemon", LOG_PID, LOG_DAEMON);
 		set_die_routine(daemon_die);
 	} else
+*/
 		/* avoid splitting a message in the middle */
 		setvbuf(stderr, NULL, _IOFBF, 4096);
 
